sim-administration/uploader/upload-sim-batch.go

```diff
@@ -102,16 +102,16 @@ func isICCID(s string) bool {
 	return match
 }
 
-func checkICCIDSyntax(name string, potentialIccid string) {
+func CheckICCIDSyntax(name string, potentialIccid string) {
```

Comment:
There is no need to export (capitalize) these functions.  All the files in a directory belong to the same package and thus have access to all names in that package.

Reply:
even when testing?

Reply:
Yep.  When testing, the package is built "augmented" with the tests, so they belong to the same package as the code.